pkg/providers/aws/provider_postgres.go

```diff
@@ -129,6 +132,32 @@ func (p *AWSPostgresProvider) CreatePostgres(ctx context.Context, pg *v1alpha1.P
 	// setup aws RDS instance sdk session
 	rdsSession := createRDSSession(stratCfg, providerCreds)
 
+	// Retrieve service maintenance updates, create and export Prometheus metrics
+	output, err := rdsSession.DescribePendingMaintenanceActions(&rds.DescribePendingMaintenanceActionsInput{})
+	if err != nil {
+		msg := "rds serviceupdates error"
+		return nil, croType.StatusMessage(msg), errorUtil.Wrap(err, msg)
+	}
+
+	logrus.Info(fmt.Sprintf("rds serviceupdates: %d available", len(output.PendingMaintenanceActions)))
+	metricName := CRO_AWS_RDS_SERVICE_MAINTENANCE
+	for _, su := range output.PendingMaintenanceActions {
+		metricLabels := map[string]string{}
+
```

Comment:
I think maybe we can add more labels here, the likes of `clusterID`, `resourceID`, `namespace`, `rdsIdentifier`. 
What do you think? we could create a more meaning full alert